### ISSUE TITLE
something

### ISSUE THREAD

**Issue by jakpra** [UniversalConceptualCognitiveAnnotation/docs#60]
What about `something` in `I was afraid of something happening to me`?

- F outside P: treating as expletive
- F inside P:  [ something_C happening_F ]_P (light-verby) or [ something_F happening_C ]_P (inverse light verb?)
- unanalyzable P [ something happening ]_P
- D outside P: ???
- `something` is P, `happening` is D

**Comment by omriabnd** [comment]
If I'm not mistaken, the position of the F doesn't matter for evaluation,
as it is often ambiguous. I would leave it outside the P since the
guidelines instruct us to prefer shorter P wherever possible.

**Comment by jakpra** [comment]
Right. But then what is the P and what the F? `happen` seems pretty light/shell-verby to me.
If it were `the accident happened`, clearly `accident` should be P. But with pronouns I'm not sure.

Cf.
`it happened`
`this happened`
where `it` and `this` can be coreferential with another, more explicit event mention.

**Comment by omriabnd** [comment]
I see what you mean. So the question is how do we annotate pronouns that refer to events. Maybe we should label them as P?

**Comment by dotdv** [comment]
Back then we discussed "what happened" and eventually decided to annotate "what" as A. We added this example to the guidelines: ``Tell_P me_A [what_C [{what}_A happened_P]_E]_A'' (appears in "Question words")
It seems similar to "I was afraid of something happening to me", no?

**Comment by omriabnd** [comment]
right. yes, let's follow the same route here.
